src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -164,5 +164,68 @@ InModuleScope Arcus.Scripting.LogicApps {
                 }
             }
         }
+        Context "Cancel Logic Apps runs" {
+            It "Cancel all running instances for a Logic App"{
+                # Arrange
+                $resourceGroupName = $config.Arcus.ResourceGroupName
+                $logicAppName = Create-AzLogicAppName
+                $workflowDefinition = '{
+                    "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+                    "actions": {
+                        "Delay": {
+                            "inputs": {
+                                "interval": {
+                                    "count": 5,
+                                    "unit": "Minute"
+                                }
+                            },
+                            "runAfter": {},
+                            "type": "wait"
+                        }
+                    },
+                    "outputs": {},
+                    "parameters": {},
+                    "triggers": {
+                        "Recurrence": {
+                            "recurrence": {
+                                "frequency": "Second",
+                                "interval": 1
+                            },
+                            "type": "recurrence"
+                        }
+                    },
+                    "contentVersion": "1.0.0.0"
+                }'
```

Comment:
```suggestion
                $workflowDefinition = '{ "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#", "actions": { "Delay": { "inputs": { "interval": { "count": 5, "unit": "Minute" } }, "runAfter": {}, "type": "wait" } }, "outputs": {}, "parameters": {}, "triggers": { "Recurrence": { "recurrence": { "frequency": "Second", "interval": 1 }, "type": "recurrence" } }, "contentVersion": "1.0.0.0" }'
```

Is the actual Logic App content important here? I would guess not, in which case we could maybe hide it in a single line?
But, if you think that it is important to kind of know 'what' you are cancelling in the test, feel free to dismiss this. Just a suggestion 😉 .

Reply:
Ok, good!